ui/vic-ui-h5c/vic/src/vic-app/src/app/vch-view/vch-view.component.spec.ts

```diff
@@ -43,40 +43,32 @@ let responseProperlyFormatted: boolean = true;
 class VicVmViewServiceStub {
     public totalVchsLength$: Observable<number>;
     private vchsSubj: Subject<VirtualContainerHost[]>;
-    public vchs$: Observable<VirtualContainerHost[]>;
     private data: VirtualContainerHost[] = [];
 
     constructor() {
         this.vchsSubj = new Subject<VirtualContainerHost[]>();
-        this.vchs$ = this.vchsSubj.asObservable();
 
         // populates data with either correctly or incorrectly formatted data
         // based on the responseProperlyFormatted flag
-        let vchResponse = responseProperlyFormatted ?
-            getVchResponseStub().results : getMalformedVchResponseStub().results;
-        for (let objId in vchResponse) {
-            if (vchResponse.hasOwnProperty(objId)) {
-                this.data.push(new VirtualContainerHost(vchResponse[objId]));
-            }
-        }
 
-        // sets up an observable for the total length of vchs
-        if (responseProperlyFormatted) {
-            this.totalVchsLength$ = new Observable<number>(observer => {
-                observer.next(this.data.length);
-            });
-        } else {
-            this.totalVchsLength$ = new Observable<number>(observer => {
-                observer.next(0);
-            });
-        }
     }
 
-    reloadVchs() {
-        if (responseProperlyFormatted) {
-            this.vchsSubj.next(this.data);
-        } else {
+    reloadVchs(): Observable<VirtualContainerHost[]> {
```

Comment:
This should be something more descriptive like getVchData. You aren't reloading anything in this method.